### PR TITLE
Added practical ES6 Vue example how to send data to PHP (and others).

### PR DESCRIPTION
**Pull request by juhasev**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,3 +38,71 @@ const formData = objectToFormData(
...
+    export default {
+
+        methods: {
+            data() {
+                return {
+                    uploading: false
+                }
+            },
+            sendMessage() {
+                import objectToFormData from 'object-to-formdata';
```
Comment (alvaro-canepa) [inline README.md]: Why import `objectToFormData` twice?

Comment (juhasev) [reply README.md]: Must be a typo :-) Import at the top should remain.

**Comment by therealparmesh** [comment]
I love the idea of having examples! But I wonder if maybe we could link to examples in CodeSandbox instead of checking more things into the repository. What so you think?

**Comment by juhasev** [comment]
@therealparmesh Don't know what you mean by CodeSandbox? I am fine either way.

**Comment by therealparmesh** [comment]
Check out https://codesandbox.io/.

**Pull request closed without merging**